features/dcs_failsafe_mode.feature

```diff
@@ -0,0 +1,72 @@
+Feature: dcs failsafe mode
+  We should check the basic dcs failsafe mode functioning
+
+  Scenario: check failsafe mode can be successfully enabled
+    Given I start postgres0
+    And postgres0 is a leader after 10 seconds
+    And I sleep for 3 seconds
+    When I issue a PATCH request to http://127.0.0.1:8008/config with {"loop_wait": 2, "ttl": 20, "retry_timeout": 5, "failsafe_mode": true}
+    Then I receive a response code 200
+    And I sleep for 3 seconds
+    When I issue a GET request to http://127.0.0.1:8008/failsafe
+    Then I receive a response code 200
+    And I receive a response postgres0 http://127.0.0.1:8008/patroni
+    When I issue a PATCH request to http://127.0.0.1:8008/config with {"postgresql": {"parameters": {"wal_level": "logical"}}}
+    Then I receive a response code 200
+    When I issue a PATCH request to http://127.0.0.1:8008/config with {"slots": {"dcs_slot_0": {"type": "logical", "database": "postgres", "plugin": "test_decoding"}}}
+    Then I receive a response code 200
+
+  @dcs-failsafe
+  Scenario: check one-node cluster is functioning while DCS is down
+    Given DCS is down
+    Then postgres0 role is the primary after 20 seconds
```

Comment:
I think here we need to have a sleep 12 before check.

Reply:
```
    Given DCS is down
    And I sleep for 12 seconds
    Then postgres0 role is the primary after 20 seconds
```
like this?

Reply:
Then postgres0 role is the primary after 10 seconds

When executed on GH Actions we use multiplier 5: https://github.com/zalando/patroni/blob/master/features/environment.py#L878
2*5 == 10s, which still might be not enough for promote to finish on MacOS. With 10s we get the timeout 50s.

Reply:
Ah, now I get it, thanks